testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergHiveTablesCompatibility.java

```diff
@@ -65,4 +65,28 @@ public void testHiveSelectFromIcebergTable()
 
         onTrino().executeQuery("DROP TABLE iceberg.default." + tableName);
     }
+
+    @Test(groups = {ICEBERG, STORAGE_FORMATS, HMS_ONLY})
+    public void testIcebergCannotCreateTableNamesakeToHiveTable()
```

Comment:
`testIcebergCreateTableWhenHiveTableExists`

Reply:
Changed to `testIcebergCreateTableWhenNamesakeHiveTableExists`